hapi-utils/src/main/java/com/hederahashgraph/fee/CryptoFeeBuilder.java

```diff
@@ -632,16 +632,16 @@ public FeeData getCryptoAddLiveHashTxFeeMatrices(TransactionBody txBody, SigValu
     long sbpr = 0;
 
     // calculate BPT - Total Bytes in Transaction
-    int txBodySize = 0;
+    long txBodySize = 0;
     txBodySize = getCommonTransactionBodyBytes(txBody);
-    bpt = txBodySize + getCryptoAddLiveHashBodyBodyTxSize(txBody) + (BASIC_ENTITY_ID_SIZE)
+    bpt = (long) txBodySize + getCryptoAddLiveHashBodyBodyTxSize(txBody) + (BASIC_ENTITY_ID_SIZE)
```

Comment:
Remove this cast since `txBodySize` was changed to `long`

Reply:
Yes changes were in my local. Fixed it. Thanks!